app/models/models.js

```diff
@@ -2166,22 +2161,22 @@ YUI.add('juju-models', function(Y) {
           var owner = units[0].service;
           var ownerIndex = owners[owner] >= 0 ? owners[owner] + 1 : 0;
           owners[owner] = ownerIndex;
-          machineName = owner + '=' + ownerIndex;
+          machineName = owner + '/' + ownerIndex;
           machineNames[machine.id] = machineName;
         }
 
         units.forEach(function(unit) {
           var serviceName = unit.service;
-          if (serviceName === owner) {
-            // Deployer doesn't allow placing units on a machine owned by a unit
-            // of the same service.
-            return;
+
```

Comment:
can you comment this a bit? I see what's up but I worry this will be more confusing in 6mo.